jwst/ramp_fitting/ramp_fit.py

```diff
@@ -416,6 +416,7 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
             segs_4[num_int, :, rlo:rhi, :] = segs_beg_3
             var_p4[num_int, :, rlo:rhi, :] = den_p3 * med_rates[ rlo:rhi, :]
             var_r4[num_int, :, rlo:rhi, :] = num_r3 * den_r3
+            var_r4[num_int,:,rlo:rhi,:] = var_r4[num_int, :, rlo:rhi,:] /gain_2d[rlo:rhi,:]**2
```

Comment:
This is fine, but you could use gain_sect**2 instead of gain_2d[rlo:rhi,:]**2.

Reply:
Minor, but for the sake of not inflating the code base, couldn't the division by gain be incorporated into the computation of `var_r4` in line 418?

Reply:
I have committed a change with to do the unit conversion on one line and add a comment.